package.json

```diff
@@ -32,14 +32,12 @@
     "eslint": "6.0.1",
     "eslint-config-prettier": "6.15.0",
     "husky": "1.3.1",
-    "istanbul": "1.1.0-alpha.1",
```

Comment:
istanbul has been deprecated for a loooong time.  It was causing issues with compiling the source to node 14 `es2020`.  This is only used for the unigration tests.  I attempted to keep coverage for a bit but that test running is bananas so I ended up just punting and removing the coverage piece of the puzzle since those tests are headed for deletion anyway.